### PR TITLE
ProgressBar: add Angular, Vue and React code snippets

### PR DESCRIPTION
**Pull request by ksercs**

### CODE REVIEW

**Review by AlexJameson — APPROVED** [review]

File: concepts/05 Widgets/ProgressBar/05 Progress Status.md [hunk 1]
```diff
@@ -1,25 +1,111 @@
...
     $(function(){
          $("#progressBarContainer").dxProgressBar({
              value: 40,
              min: 0,
              max: 100,
              showStatus: true,
-             statusFormat: function(ratio, value) {
-                 return globalizeFormatter(ratio) + "%";
+             statusFormat: function(value) {
+                 return value + "USD";
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/ProgressBar/05 Progress Status.md]:
```suggestion
                 return value + " USD";
```

File: concepts/05 Widgets/ProgressBar/05 Progress Status.md [hunk 2]
```diff
@@ -1,25 +1,111 @@
...
+        [showStatus]="true"
+        [statusFormat]="statusFormat">
+    </dx-progress-bar>
+
+    <!--TypeScript-->
+    import { DxProgressBarModule } from "devextreme-angular";
+    // ...
+    export class AppComponent {
+        statusFormat(value) {
+            return value + "USD";
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/ProgressBar/05 Progress Status.md]:
```suggestion
            return value + " USD";
```

File: concepts/05 Widgets/ProgressBar/05 Progress Status.md [hunk 3]
```diff
@@ -1,25 +1,111 @@
...
+
+    import { DxProgressBar } from 'devextreme-vue/progress-bar';
+
+    export default {
+        components: {
+            DxProgressBar
+        },
+        methods: {
+            statusFormat(value) {
+                return value + "USD";
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/ProgressBar/05 Progress Status.md]:
```suggestion
                return value + " USD";
```

File: concepts/05 Widgets/ProgressBar/05 Progress Status.md [hunk 4]
```diff
@@ -1,25 +1,111 @@
...
+
+    import { ProgressBar } from 'devextreme-react/progress-bar';
+
+    class App extends React.Component {
+        constructor(props) {
+            super(props);
+        }
+
+        statusFormat(value) {
+            return value + "USD";
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/ProgressBar/05 Progress Status.md]:
```suggestion
            return value + " USD";
```

**Review by RomanTsukanov — CHANGES_REQUESTED** [review]

File: concepts/05 Widgets/ProgressBar/05 Progress Status.md (hunk 1 above)
Comment (ksercs) [reply concepts/05 Widgets/ProgressBar/05 Progress Status.md]: fixed

File: concepts/05 Widgets/ProgressBar/05 Progress Status.md (hunk 2 above)
Comment (ksercs) [reply concepts/05 Widgets/ProgressBar/05 Progress Status.md]: fixed

File: concepts/05 Widgets/ProgressBar/05 Progress Status.md (hunk 3 above)
Comment (ksercs) [reply concepts/05 Widgets/ProgressBar/05 Progress Status.md]: fixed

File: concepts/05 Widgets/ProgressBar/05 Progress Status.md (hunk 4 above)
Comment (ksercs) [reply concepts/05 Widgets/ProgressBar/05 Progress Status.md]: fixed

**Review by RomanTsukanov — APPROVED** [review]

**Pull request merged**